parm/archive/gefs_atmos.yaml.j2

```
@@ -0,0 +1,33 @@
+{% set cycle_HH = current_cycle | strftime("%H") %}
+{% set cycle_YMD = current_cycle | to_YMD %}
+{% set cycle_YMDH = current_cycle | to_YMDH %}
+{% set head = RUN + ".t" + cycle_HH + "z." %}
+
+gefs:
+    name: "GEFS"
+    target: "{{ ATARDIR }}/{{ cycle_YMDH }}/gefs_atmos1.tar"
+    required:
+{% set file_set = [] %}
+{% for mem in range(1, NMEM_ENS + 1) %}
+    {% for res in ['0p25', '0p50', '1p00'] %}
+        {% set tmpl_dict = ({ '${ROTDIR}':ROTDIR,
+                              '${RUN}':RUN,
+                              '${YMD}':cycle_YMD,
+                              '${HH}':cycle_HH,
+                              '${GRID}':res,
+                              '${MEMDIR}':'mem%03d'|format(mem) }) %}
+
+        {% set COMIN_ATMOS_GRIB = COM_ATMOS_GRIB_GRID_TMPL | replace_tmpl(tmpl_dict) %}
+
+        # Select pgrb files to copy to the atardir
+        {% if RUN == "gefs" %}
+            {% if path_exists(COMIN_ATMOS_GRIB) %}
+                {% for fhr in range(FHMIN_GFS, FHMAX_GFS + FHOUT_GFS, FHOUT_GFS) %}
+                    {% set file_set = COMIN_ATMOS_GRIB | relpath(ATARDIR) ~ "/" ~ head ~ "pgrb2b." ~
```

The `relpath` call here isn't doing anything because `ATARDIR` is outside of the `COM` tree.  The call on line 28 should be sufficient.
```suggestion
                    {% set file_set = COMIN_ATMOS_GRIB ~ "/" ~ head ~ "pgrb2b." ~
```

---

sorry test file was uploaded